CMakeLists.txt

```diff
@@ -154,6 +155,9 @@ if (BUILD_STATIC)
     set_target_properties(redisearch PROPERTIES LINKER_LANGUAGE CXX)
     setup_shared_object_target(redisearch "")
     target_link_libraries(redisearch redisearch-geometry VectorSimilarity ${REDISEARCH_LIBS} ${CMAKE_LD_LIBS})
+   	target_link_libraries(redisearch
+  		debug ${root}/src/redisearch_rs/target/debug/libtrie_rs.a
+  		optimized ${root}/src/redisearch_rs/target/release/libtrie_rs.a)
```

Comment:
If you link to this path, why you also copy it to the main bin directory? Am I missing something?

Reply:
I added the copy later on as a way to have all final artefacts in the "expected" location, but I forgot to change the linking directory.
I have amended it in [1068b0f](https://github.com/RediSearch/RediSearch/pull/5694/commits/1068b0f504dfa96408759c8e3ef239f77a5e85a4) to have all artefacts in the bin dir, and link from there.